srai/embedders/__init__.py

```diff
@@ -1,8 +1,17 @@
 """Embedders."""
 
-from ._base import Embedder
-from .count_embedder import CountEmbedder
+# Force import of required base classes
+from srai.embedders._base import Embedder
+from srai.embedders.count_embedder import CountEmbedder
+
+from .contextual_count_embedder import ContextualCountEmbedder
 from .gtfs2vec_embedder import GTFS2VecEmbedder
 from .highway2vec import Highway2VecEmbedder
```

Comment:
Why some imports are relative and some absolute?

Reply:
I had to force `CountEmbedder` to be imported before `ContextualCountEmbedder` since the second one relies on the first, but `isort` rules change the order to be alphabetical. Couldn't do it using `noqa` comments and IDE changed sort order all the time.

Reply:
Why does it happen? There is no import of `CountEmbedder` in contextual file?

Reply:
The is, it's a subclass of `CountEmbedder` just like `Hex2VecEmbedder`. If we leave it ordered alphabetically, the interpreter throws an error that the module is partially loaded, since the child is imported before the parent.

Reply:
I'll change imports inside `ContextualCountEmbedder` to avoid using imports from upper module, but import directly from the `CountEmbedder` file.

Reply:
Ok, now I get where this error came from. Thanks!